extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -695,11 +742,10 @@ angular.module('oppia').component('svgFilenameEditor', {
       var createBezierControlPoints = function(left, top) {
         // This function is used to add the control points for the quadratic
         // bezier curve which is used to control the position of the curve.
-        var size = ctrl.fabricjsOptions.size;
         var circle = new fabric.Circle({
           left: left,
           top: top,
-          radius: parseInt(size.substring(0, size.length - 2)) + 2,
+          radius: getStrokeWidth() + 2,
```

Comment:
Why +2? Also, why is getStrokeWidth used to calculate radius?

Reply:
Done.